src/videojs.ads.js

```diff
@@ -155,11 +160,43 @@ var
         src: player.currentSrc(),
         currentTime: player.currentTime(),
         type: player.currentType()
-      };
+      },
+      tracksPlayingHandler;
+
+    tracksPlayingHandler = function() {
+      var trackEls = this.el().querySelector('.vjs-tech').querySelectorAll('track'),
+          track,
+          i = trackEls.length,
+          j = 0;
+
+      while (i--) {
+        track = trackEls[i].track;
+
+        for (j = 0; j < suppressedTracks.length; j++) {
+          if (suppressedTracks[j].track === track) {
+            suppressedTracks[j].mode = track.mode;
+          }
+        }
+
+        track.mode = 'disabled';
```

Comment:
Two questions: 

I don't think `getPlayerSnapshot()` should be making side-effecting changes to the player (or in this case, setting event listeners that create side-effecting changes). Can this be moved into the transitions into and out of linear ad mode?

This loop seems to unconditionally disable the track, whether it is on the suppressed list or not. Is that correct? Should it be?